flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1308,7 +1310,7 @@ def _has_file_with_content(self, filename, content):
             test_mountpoint.remove()
         return result
 
-    def invariant(self, case, current_local_state):
+    def invariant(self, case, test_objects):
         """
         An invariant that should always hold. Specifically, after the path to a
```

Comment:
:question: I think this first sentence is redundant. Maybe delete it. Maybe rename from `invariant` (which doesn't help), to `writes_persist_or_fail`.

OTOH, this patch is already rather large, so maybe don't bother.